generate_outputs.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import json
 import os
+import random
```

Comment:
Is this used? I don't see it in the diff

Reply:
ping

Reply:
removed